spring-integration-r2dbc/src/test/java/org/springframework/integration/r2dbc/inbound/R2dbcMessageSourceTests.java

```diff
@@ -89,6 +89,12 @@ public void setup() {
 				.verifyComplete());
 	}
 
+	@Test
+	public void validateComponentType() {
+		assertThat(this.defaultR2dbcMessageSource.getComponentType()).isEqualTo("r2dbc:inbound-channel" +
+				"-adapter");
```

Comment:
Isn't that would be better to wrap the line on the `.isEqualTo()` instead of splitting string?